module/rdpCursor.c

```diff
@@ -280,13 +280,12 @@ rdpSpriteSetCursorCon(rdpClientCon *clientCon,
     client_max_height = 32;
     sending_bpp = 0;
     can_do_new = clientCon->client_info.pointer_flags & 1;
-    can_do_large = 0;
-#if defined(CLIENT_INFO_CURRENT_VERSION) && defined(LARGE_CURSOR_FLAG_96x96)
-#if CLIENT_INFO_CURRENT_VERSION >= 20230423
+#if (CLIENT_INFO_CURRENT_VERSION >= 20230423) && defined(LARGE_CURSOR_FLAG_96x96)
```

Comment:
Sorry - should have mentioned. If CLIENT_INFO_CURRENT_VERSION >= 20230423, you probably don't need to bother with the `defined(LARGE_CURSOR_FLAG_96x96)` check as that's implicit.

Reply:
Actually, I added LARGE_CURSOR_FLAG_96x96 a couple of days later.  I'll update that.  So we use CLIENT_INFO_CURRENT_VERSION for all the installed headers, not just struct xrdp_client_info.  Good to know.  When I change it, I'll add a comment.